src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    private String name;
+
+    public Bulldozer(String name) {

```

Comment:
Remove constructor and "name" field to be able to create instances without providing a name